ArchiSteamFarm/ArchiSteamFarm.csproj

```diff
@@ -51,11 +51,12 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="AngleSharp" Version="0.13.0" />
```

Comment:
We can get rid of this, `AngleSharp.XPath` already depends on it, the less entries the better.